controllers/managedocs_controller_test.go

```diff
@@ -536,6 +531,22 @@ var _ = Describe("ManagedOCS controller", func() {
 					Expect(k8sClient.Delete(ctx, secret)).Should(Succeed())
 				})
 			})
+			When("there is no onboarding-validation-key field in the add-on parameters secret", func() {
```

Comment:
Where is the opposite test that verifies successful creation with correct content?